docs/general/attributes.md

```diff
@@ -306,11 +306,12 @@ Examples of `peer.service` that users may specify:
 
 ## General identity attributes
 
-These attributes may be used for any operation with an authenticated and/or authorized enduser.
+These attributes may be used for any operation with an authenticated or anonymous enduser.
 
 <!-- semconv identity -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `enduser.anon_id` | string | Randomly generated id of an anonymous user that is persisted throughout the lifetime of the visits. | `V1StGXR8_Z5jdHi6B-myT` | Recommended |
```

Comment:
just wanted to raise the question of `anon_id` vs `anonymous_id`, I'm not sure if we have good (any?) guidance on when to use abbreviations in attribute names

Reply:
(btw I opened #602 to seek more comments / guidance on the general question of abbreviations)